From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Biswapriyo Nath <nathbappai@gmail.com>
Date: Fri, 3 Dec 2021 14:19:32 +0000
Subject: [PATCH 1/1] tabs: Fix build with clang in mingw

Include stdlib.h for qsort function declaration. This fixes the error:
implicit declaration of function 'qsort' is invalid in C99

diff --git a/pango/pango-tabs.c b/pango/pango-tabs.c
index 1111111..2222222 100644
--- a/pango/pango-tabs.c
+++ b/pango/pango-tabs.c
@@ -22,6 +22,7 @@
 #include "config.h"
 #include "pango-tabs.h"
 #include "pango-impl-utils.h"
+#include <stdlib.h>
 #include <string.h>
 
 typedef struct _PangoTab PangoTab;
